src/main/java/uk/gov/justice/hmiprobation/casesampler/services/RoAllocationAdjuster.kt

```diff
@@ -0,0 +1,104 @@
+package uk.gov.justice.hmiprobation.casesampler.services
+
+import org.slf4j.LoggerFactory
+import uk.gov.justice.hmiprobation.casesampler.utils.Counter
+import uk.gov.justice.hmiprobation.casesampler.utils.Result
+import uk.gov.justice.hmiprobation.casesampler.utils.Size
+import uk.gov.justice.hmiprobation.casesampler.utils.Size.Adjusted
+import uk.gov.justice.hmiprobation.casesampler.utils.Size.SampleSize
+
+data class RoSize(val ro: String, val size: Size) {
```

Comment:
Interesting... could you have extended Pair but still have a data class or is that not possible?

Reply:
That would be good but unfortunately Pair is a `data class` so final. It's a shame you can't treat a two field data class as a pair - in a lot of places it doesn't matter (when destructuring) - its just when converting to/from maps where its a bit of a pain.